ocs_ci/deployment/deployment.py

```diff
@@ -3411,7 +3431,17 @@ def deploy(self):
         for i in acm_indexes:
             config.switch_ctx(i)
             self.create_dpa(self.meta_obj.bucket_name)
+
```

Comment:
Is OADP operator installation is covered here?

Reply:
Yes it's covered